src/movies.js

```diff
@@ -1,28 +1,76 @@
+//let fs = require("fs");
+const movies = require("./data");
+//let Data = fs.readFileSync("./data.js", "utf-8");
+
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors() {
+  let Directors = []
+  movies.forEach(element => {
+    Directors.push(element.Directors)
+  });
```

Comment:
This solution works! But please try solving this this iteration using `map() `method